README.md

```
@@ -68,28 +68,12 @@ composer global require koriym/xdebug-mcp
 
 ### 3. Setup AI Integration
 
-#### Option A: Skills (Recommended for Claude Code / Codex)
-
-```bash
-ln -s ~/.composer/vendor/koriym/xdebug-mcp/skills/xdebug ~/.claude/skills/xdebug
+**Claude Code:**
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)